lib/comfortable_mexican_sofa/seeds.rb

```diff
@@ -26,8 +26,9 @@ def initialize(from, to = from)
       end
     end
 
-    def import!
-      %w[Layout Page Snippet File].each do |klass|
+    def import!(klasses = nil)
+      klasses = klasses || %w[Layout Page Snippet File]
```

Comment:
you can define it as `import!(classes = %w[Layout Page Snippet File])`. `classes` is fine, it's not a reserved word.